LabDrivers/Tool.py

```diff
@@ -318,16 +330,26 @@ def ask(self, msg, num_bytes=None):
 
                 try:
 
-                    answer = self.connection.ask(msg)
-
+                    answer = self.connection.query(msg) #ask is depricated
```

Comment:
```suggestion
                    answer = self.connection.query(msg) #ask is deprecated
```

Can you provide links to back your claim (not that I don't trust you, but it is good practice)?